libraries/AP_NavEKF2/AP_NavEKF2_core.h

```diff
@@ -359,6 +359,11 @@ class NavEKF2_core : public NavEKF_core_common
     // request a reset the yaw to the EKF-GSF value
     void EKFGSF_requestYawReset();
 
+    // return true if we are aligned, so error score is valid
+    bool aligned(void) const {
+        return yawAlignComplete && tiltAlignComplete;
```

Comment:
Yaw align complete will be false until after launch for flight without a magnetometer when using EKF-GSF fr yaw

Reply:
right, so this will prevent us lane switching until after launch. We certainly can't do error score based switching with the current scoring system until then anyway as the error score is always zero unless we have both alignments.
We could instead use a different scoring system when we don't have a yaw alignment, and I can implement that if you prefer. We'd just need to be careful not to mix the two scores.